readme.md

```diff
@@ -1,77 +1,30 @@
 **BASED ON [firesphere/solr-search](https://codeberg.org/Firesphere/silverstripe-solr)**
 
-[![Maintainability](https://api.codeclimate.com/v1/badges/55c8967ef25e37182e3d/maintainability)](https://codeclimate.com/github/Firesphere/silverstripe-solr-search/maintainability)
-[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/Firesphere/silverstripe-solr-search/badges/quality-score.png?b=primary)](https://scrutinizer-ci.com/g/Firesphere/silverstripe-solr-search/?branch=primary)
-[![Code Coverage](https://scrutinizer-ci.com/g/Firesphere/silverstripe-solr-search/badges/coverage.png?b=primary)](https://scrutinizer-ci.com/g/Firesphere/silverstripe-solr-search/?branch=primary)
-[![CircleCI](https://circleci.com/gh/Firesphere/silverstripe-solr-search/tree/primary.svg?style=svg)](https://circleci.com/gh/Firesphere/silverstripe-solr-search/tree/primary)
-[![codecov](https://codecov.io/gh/Firesphere/silverstripe-solr-search/branch/primary/graph/badge.svg?token=BACi0PeDRD)](https://codecov.io/gh/Firesphere/silverstripe-solr-search)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/55c8967ef25e37182e3d/test_coverage)](https://codeclimate.com/github/Firesphere/silverstripe-solr-search/test_coverage)
-[![Build Status](https://scrutinizer-ci.com/g/Firesphere/silverstripe-solr-search/badges/build.png?b=primary)](https://scrutinizer-ci.com/g/Firesphere/silverstripe-solr-search/build-status/primary)
-[![Support us](https://enjoy.gitstore.app/repositories/badge-Firesphere/silverstripe-solr-search.svg)](https://enjoy.gitstore.app/repositories/Firesphere/silverstripe-solr-search)
-
 # License
 
 [LGPL V3](LICENSE.md)
 
-
-[![ko-fi](https://www.ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/B0B11GKLY) 
-
 ## Disclaimer
 
 If this module breaks your website, you get to keep all the pieces.
 
 # Modern SilverStripe Solr Search
 
-[Full documentation](https://firesphere.github.io/solr-docs/) or see the [docs folder](docs/index.md). Please read the documentation before asking questions.
+See the [docs folder](docs/index.md). Please read the documentation before asking questions.
 A lot of the questions are answered by reading the documentation.
 
 ## Solarium documentation:
 
 https://solarium.readthedocs.io
 
-## API Docs:
-
-https://firesphere.github.io/solr-api/
-
-## Usage and installation docs:
-
-https://firesphere.github.io/solr-docs/
-
 ## Supports
 
-Solr4 backward compatibility is available, default support is
-Solr8
+Solr4 backward compatibility is available
+Default support is Solr8
+Solr9 is supported
```

Comment:
This is a bit difficult to read as it's formatted as one long sentence. See https://github.com/signify-nz/silverstripe-solr/tree/docs/publishable-docs